conf/messages.en

```diff
@@ -640,7 +640,7 @@ declaration.transportInformation.active.help-item1 = The details of the transpor
 declaration.transportInformation.active.help-item2 = The country the transport is registered in, go to data element 7/15 Nationality of active means of transport crossing the border.
 
 
-declaration.transportInformation.transportPayment.paymentMethod = If this is an express consignment, tell us how you paid for the transport
+declaration.transportInformation.transportPayment.paymentMethod = How is the transport paid for?
```

Comment:
I changed this recently.   The previous title is correct I believe - see https://exports-declaration-prototype.herokuapp.com/version31/dev-ready/active-transport-payment-method?what-declaration-type=standard

Reply:
Then again, you have a screenshot on your ticket.  I'd check with design